getting-started/cli/recipes.md

```diff
@@ -0,0 +1,152 @@
+This stores some metadata for the Amber CLI to make use of for generating migrations,
+scaffolding templates, and more.  The recipe name that you use to generate the application is saved
+in the `.amber.yml` file so that when you generate controllers, models and views with the
+scaffold generator the same recipe is used - you don't have to specify the recipe each time.
+
+### `amber new`
+
+```shell
+amber new NAME OPTIONS
+
+Arguments:
+  NAME  name/path of project
+
+Options:
+  -d      Select the database database engine, can be one of: pg | mysql | sqlite (default: pg)
+  --deps  Installs project dependencies, this is the equivalent of running (shards update)
+  -t      Selects the template engine language, can be one of: slang | ecr (default: slang)
+  -r      Use a named recipe.  See documentation at https://amberframework.org.
+```
+
+## Example Usage
+Using `amber new microsecond-blog -r damianham/modular` will generate a skeleton
+Amber application in `./microsecond-blog` using the _modular_ recipe that was
+provided by the github user _damianham_.
+
+You can have a running web application in a matter of minutes:
+1. `amber new microsecond-blog -r damianham/modular`
+1. `cd microsecond-blog`
+1. `shards install`
+1. `amber run &` or `amber watch &` (also can execute these in foreground of separate terminal)
+1. `open http://localhost:3000`
+
+Full example in terminal:
+```shell
+$ amber new microsecond-blog -r damianham/modular
+Rendering App microsecond-blog in ./microsecond-blog
+new       .amber.yml
+new       .amber_secret_key
+new       .gitignore
+new       .travis.yml
+new       config/application.cr
+new       config/database.yml
+... [clipped]
+new       src/views/layouts/_nav.slang
+new       src/views/layouts/application.slang
+new       src/microsecond-blog.cr
+$ cd microsecond-blog/
+$ shards install
+Updating https://github.com/amberframework/amber.git
+Updating https://github.com/luislavena/radix.git
+... [clipped]
+Installing garnet_spec (0.1.1)
+Installing selenium (0.3.0)
+$ amber run &
+[1] 65034
+💎  Crystalizing...
+microsecond-blog
+$ 💎  Crystalization complete!
+I, [2017-11-14 10:05:58 -0800 #65094]  INFO -- : [Amber 0.3.0] serving application "microsecond-blog" at http://0.0.0.0:3000
+I, [2017-11-14 10:05:58 -0800 #65094]  INFO -- : Server started in development.
+I, [2017-11-14 10:05:58 -0800 #65094]  INFO -- : Startup Time 00:00:00.0013140
+$ open http://localhost:3000
+200  | GET  "/"  | 294.0µs
+Params:
+```
+
+# Amber Generate Command
+
+The generators in amber are a great way to get an application up and running quickly.
+In addition, they help keep code consistent and following convention.  See the
+**Generate** option of the **Command Line Tool** for details about the generate command.
+
+Once you have generated an application with a recipe, the same recipe will be used
+by the generate command when you generate controllers, models or scaffold controllers,
+models and views with the scaffold generator.
```

Comment:
How about: 

```diff
- Once you have generated an application with a recipe, the same recipe will be used by the generate command when you generate controllers, models or scaffold controllers, models and views with the scaffold generator.
+ Once an application is generated with a recipe, it will be used by future generate or scaffold commands to maintain consistency with the recipe. For example, an application generated with a "json-api" recipe will continue to generate controllers and views according to the "json-api" recipe.
```